app/src/main/res/values/colors.xml

```diff
@@ -27,13 +27,12 @@
 
     <color name="pocket_onboarding_background">#e60c0c0d</color>
 
-    <color name="button_highlight">#64ffffff</color>
-    <color name="button_default">@color/translucent_tv_ink</color>
-    <color name="button_disabled">@color/tv_ink</color>
-    <color name="button_focused">@color/tv_blue50</color>
+    <color name="button_highlight">#00ffffff</color>
```

Comment:
#00xxxxxx just means that the color is transparent, because it's at 00 transparency. If a hex color has 8 characters, the first two indicate the transparency. If there are only 6, then the transparency is assumed to be FF or 100%.